lib/zen_to_i/constant.rb

```diff
@@ -2,9 +2,12 @@
 class String
   module Zen2Num
     KANNUM = { '〇' => 0, '' => 1,'一' => 1, '二' => 2,'三' => 3, '四' => 4,
-      '五' => 5, '六' => 6, '七' => 7, '八' => 8, '九' => 9 }
-    KANORDER = {'十' => 10,'百' => 100, '千' => 1000 , '' => 1}
-    KANORDER_F = { '万' => 10**4 , '億' => 10**8, '兆' => 10**12 , ''=> 1}
+      '五' => 5, '六' => 6, '七' => 7, '八' => 8, '九' => 9,
+      '零' => 0, '壱' => 1, '弐' => 2, '参' => 3, '肆' => 4,
```

Comment:
__[rubocop]__
* Align the elements of a hash literal if they span more than one line.